### PR TITLE
Add sig-transaction's reviewers

### PR DESCRIPTION
**Pull request by ekexium**
Add reviewers from the sig-transaction to the TiDB team: @ekexium, @longfangsong, @you06, which is missing in #522

### CODE REVIEW

**Comment by tisonkun** [comment]
@ekexium thanks for creating this PR. I'd suggest you create a vote entry under the directory https://github.com/pingcap/community/tree/master/votes and elaborate nominees' contribution (of course, previous sig-transaction's reviewers is enough, but better to have more details on their work yet).

See also the [template](https://github.com/pingcap/community/blob/master/votes/TEMPLATE.md) and https://github.com/tikv/community/pull/139/files#diff-2501407e9d546f432e26d56687cdf445fd1b9798b541cbeeeaee491b00a44fd3

File: votes/0544-sig-txn-reviewer-to-tidb.md
Comment (tisonkun) [inline votes/0544-sig-txn-reviewer-to-tidb.md:23]:
```suggestion
* Team TiDB
```

Also you're supposed to ask for [TiDB team maintainer](https://github.com/pingcap/community/blob/a00dfc32b940d4e1660dd97e65eda0acf54a40e3/teams/tidb/membership.json#L6-L30) to approve this vote.

File: teams/tidb/membership.json
Comment (tisonkun) [inline teams/tidb/membership.json:98]: > Items of lists in this file are in alphabetical order. Please keep it on modification

Please try to sort the members.

**Comment by you06** [comment]
We just move the reviewers from [tikv/community](https://github.com/tikv/community/) to here, and we three were promoted as reviewers in that place. Do we need to create vote entry here?

https://github.com/tikv/community/blob/6e036b68e1d9f27b264895512219dd0c0804f68a/sig/transaction/membership.json#L53-L63

**Comment by tisonkun** [comment]
@you06 actually I call for the migration in #522 multiple times but there is no suggestion on this memberships.

It's ok to have some information catch up later but missing the migration window I'd prefer you guys join the team via normal process, i.e., a vote.

As mentioned here https://github.com/tikv/community/pull/142#issuecomment-913739827, we tend to avoid exceptions. And a nomination for reviewers, with approvals, last for 72 hours. It won't take a long time.

**Review by jackysp — APPROVED** [review]
LGTM

File: votes/0544-sig-txn-reviewer-to-tidb.md
Comment (tisonkun) [inline votes/0544-sig-txn-reviewer-to-tidb.md:19]:
```suggestion
The vote will be open for at least 3 days unless there is an objection or not enough votes.
```

according to https://github.com/pingcap/community/blob/master/teams/README.md#actions

**Comment by cfzjywxk** [comment]
LGTM

**Review by MyonKeminta — APPROVED** [review]
I didn't understand it well. How can I vote? Is an approval to this PR enough?

**Comment by tisonkun** [comment]
@MyonKeminta thanks for showing you approval. According the [governance rules](https://github.com/pingcap/community/tree/master/teams#voting) accepted earlier #522 

> Decisions regarding the project are made by votes on the community repository. Votes are clearly indicated by a pull request adding an entry under votes folder. Votes may contain multiple items for approval and these should be clearly separated. Voting is carried out by replying to the vote pull request. Voting may take three flavors


You can show your approval by GitHub out-of-the-box approval.

> For team decisions, only the votes of active team maintainers are binding. Non-binding votes are still useful for those with binding votes to understand the perception of an action in the wider community.

So only @pingcap/tidb-maintainers has binding votes to this decision process. Only binding vote counts.

For promote a new reviewer, we require a minimal length on 3 days so with 2 approval from @jackysp & @cfzjywxk , this votes will be closed as accepted tomorrow if there is no further objections.

File: votes/0544-sig-txn-reviewer-to-tidb.md
Comment (tisonkun) [inline votes/0544-sig-txn-reviewer-to-tidb.md:27]:
```suggestion
Approved by 2 binding votes:

* ekexium (non-binding)
* jackysp (binding)
* cfzjywxk (binding)
* MyonKeminta (non-binding)
```

@cfzjywxk after 3 days last and there is no valid veto, you could merge this PR later today, with collected the result - I've already create a suggestion for you above.

**Comment by tisonkun** [comment]
@cfzjywxk I think you should sign CLA for this repo 😂

**Comment by tisonkun** [comment]
@ekexium @you06 @longfangsong I've added you in the tidb-reviewers team.

**Pull request merged**
